### PR TITLE
Add display name

### PR DESCRIPTION
**Pull request by iwahbe**

### CODE REVIEW

File: provider/cmd/pulumi-resource-aws/schema.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "name": "aws",
+    "displayName": "AWS",
```
Comment (flostadler) [inline provider/cmd/pulumi-resource-aws/schema.json:3]: I assume aws-native is gonna be called `AWS Cloud Control`?

**Review by flostadler — APPROVED** [review]

**Comment by flostadler** [comment]
@iwahbe you have to rebuild the minimal schema as well (`make minimal_schema`)

File: provider/cmd/pulumi-resource-aws/schema.json (hunk 1 above)
Comment (iwahbe) [reply provider/cmd/pulumi-resource-aws/schema.json:3]: That's what it's called now:

https://github.com/pulumi/registry/blob/eb8278beb938bedbac4f28c911a1cd50644fa2fb/tools/resourcedocsgen/pkg/lookup.go#L131

**Comment by t0yv0** [comment]
@corymhall can we get this one in here, I think @iwahbe move don. Thanks a ton!

**Pull request merged**